simple-test-cases/openmp_section1.c

```diff
@@ -0,0 +1,29 @@
+///TAFFO_TEST_ARGS -fopenmp
+#include <stdio.h>
+
+#define N (100)
+
+int main(int argc, char *argv[]) {
+  float foo __attribute((annotate("target('foo') scalar(range(0,10) final)"))) =
+      1.0f;
+  float pop
+      __attribute((annotate("target('pop') scalar(range(0,2000) final)"))) =
+          363;
+  float result
+      __attribute((annotate("target('result') scalar(range(0,2000)) final)"))) =
+          0;
```

Comment:
These look ugly but I'm not sure if the alternatives are prettier or not. In doubt, leave them as they are.

Reply:
I tried to use better-named variables 👌 to make the test more understandable

Reply:
I was actually referring to the indentation... But whatever, it's not an important issue, especially in tests.